frontend/src/hooks/modal/useAppNavigateCallback.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
+
+import { useCallback } from "react";
+import { AppModalType } from "../../types/AppModalType";
+import { AppModalService } from "../../services/AppModalService";
+import { ButtonClickCallback } from "../../fi/hg/frontend/components/button/Button";
+import { LogService } from "../../fi/hg/core/LogService";
+import {useNavigate} from "react-router-dom";
+
+const LOG = LogService.createLogger('useAppCallback');
+
+/**
+ * Can be used to create a callback function to open a specific modal
+ *
+ * @param modal
+ * @param id
+ */
+export function useAppNavigateCallback (
```

Comment:
The name and documentation of this callback is misleading... 

It changes a modal and then navigates to `-1`. What does that even mean? It should be documented in the JSDoc.